Tools/gsc/finalize_manifests.py

```diff
@@ -40,7 +40,7 @@ def generate_library_paths():
     # indentation.
     ld_paths = (line for line in ld_paths if not re.match(r'(^\s)', line))
 
-    return ''.join(ld_paths)
+    return ''.join(ld_paths) + os.getenv('LD_LIBRARY_PATH')[1:]
```

Comment:
Please check if `os.getenv('LD_LIBRARY_PATH')[1:]` is `None`. Application Dockerfile may not have added `LD_LIBRARY_PATH` environment variable. If its not define this will return below error.

```
Traceback (most recent call last):
  File "/finalize_manifests.py", line 155, in <module>
    main(sys.argv)
  File "/finalize_manifests.py", line 101, in main
    library_paths = generate_library_paths()
  File "/finalize_manifests.py", line 43, in generate_library_paths
    return ''.join(ld_paths) + os.getenv('LD_LIBRARY_PATH')[1:]
TypeError: 'NoneType' object is not subscriptable
```